Scripts/build.swift

```diff
@@ -46,18 +46,18 @@ enum TaskError: Error {
 }
 
 enum Platform: String, CustomStringConvertible {
+    case iOS_15
```

Comment:
Let's not add 15 yet, since we aren't actually running it here. Better to have one update that adds iOS 15 coverage than slowly add it over multiple PRs and lose track of what's supported and what's not.